Main.py

```diff
@@ -167,10 +167,11 @@ def main(args, seed=None, baked_server_options: Optional[Dict[str, object]] = No
 
     # remove starting inventory from pool items.
     # Because some worlds don't actually create items during create_items this has to be as late as possible.
-    if any(world.start_inventory_from_pool[player].value for player in world.player_ids):
+    if any(getattr(world.worlds[player].options, "start_inventory_from_pool", StartInventoryPool({})).value for player in world.player_ids):
```

Comment:
This looks strange. I wonder why `StartInventoryPool` isn't in `PerGameCommonOptions`.

Reply:
isn't the from_pool the optional one (that is opt-in per world)?

Reply:
We already merged something like this now. I suggest to look for a nicer solution at some point in the future. Maybe have a notion of a disabled option that can be enabled per world if we want to stick with static field name, or instead of using the name we could scan for field type instead.